database/dataDefinitionQueries.sql

```diff
@@ -91,3 +92,10 @@ CREATE TABLE `UserPet` (
   CONSTRAINT `UserID_fk2` FOREIGN KEY (`UserID`) REFERENCES `User` (`UserID`) ON DELETE CASCADE,
   CONSTRAINT `PetID_fk2` FOREIGN KEY (`PetID`) REFERENCES `Pet` (`PetID`) ON DELETE CASCADE
 ) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+-- Create PetBreed table
+CREATE TABLE `PetBreed` (
```

Comment:
Creating a table is good if we have a lot of data to save and if we don't pull the data a lot from the fronded. It seems like we are calling `getBreeds()` only once from the front end. I agree to use a table here.